sources/views/grid_view.js

```diff
@@ -1,67 +1,82 @@
 import {JetView} from "webix-jet";
 import {textValidation} from "models/validate_function";
 import {scrollToLastAddedElemen} from "models/scroll_function";
+
 export default class GridView extends JetView {
 	constructor(app, data){
 		super(app);
 		this.data = data;
 	}
+
 	config(){
-		var add_new_button = {
+		const _ = this.app.getService("locale")._;
+
+		const add_new_button = {
 			view:"button",
-			value:"Add new",
+			value:_("Add new"),
 			css:"webix_primary",
-			click:()=>{
-				webix.message({text:"New element was added", expire:350});
-				this.grid.add({Name:""});
-				scrollToLastAddedElemen(this.grid,true,true);
-			}
+			click:() => this.addNewElement()
 		};
-		var datatable = {
+
+		const datatable = {
 			view:"datatable",
 			select:true,
 			scrollY:true,
 			minWidth:400,
 			columns:[
-				{ id:"Name", header:"Name", editor:"text", fillspace:true, sort:"text"},
+				{ id:"value", header:_("Name"), editor:"text", fillspace:true, sort:"text"},
 				{ id:"del", header:"", template:"{common.trashIcon()}", width:60}
 			],
-			onClick:{
-				"wxi-trash":function(e, id){
-					webix.confirm({
-						title:"Country"+" would be deleted",
-						text:"Do you still want to continue?",
-						type:"confirm-warning"
-					}).then(() => {
-						webix.message({
-							text:"Element was deleted",
-							type:"info"
-						});
-						this.remove(id);
-						return false;
-					},
-					function(){
-						webix.message("Rejected");
-					}
-					);
-				}
-			},
+			onClick:{ "wxi-trash":(e,id) => this.removeElement(id) },
 			on:{
 				onValidationError:function(id){
 					this.unselect(id);
 				}
 			},
 			editable:true,
 			rules:{
-				Name:function(value){
+				value:function(value){
 					return textValidation(value,20);
 				}
 			}
 		};
+
 		return {margin:3, rows:[add_new_button,datatable]};
 	}
+
+	addNewElement(){
+		const _ = this.app.getService("locale")._;
+		
+		webix.message({text:_("New element was added"), expire:350});
+		this.data.add({value:"New"});
+		scrollToLastAddedElemen(this.grid,true,true);
```

Comment:
I do not really like this function, as it relies on the order (`getLastId()`)
you can pass id that is returned by the `.add()` method

Reply:
I took it into account

Reply:
just a small thing 

you pass id to the method and inside it check whether there is an id (and if there's no id, you do nothing). 
if is better to check id before method call `if(id) scrollTo..`, but it does not make sense in your case 

so you can just remove the check of id inside the method

Reply:
ok